CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
 
 _**Note:** Yet to be released changes appear here._
 
+## 11.14.0
```

Comment:
Why not 11.13.1? It is a bug fix after all.

Reply:
Too early in the morning I guess. 😓